doc/non_dev_deployments.md

```diff
@@ -42,9 +42,10 @@ for the public facing endpoints, instead of using self signed certificates.
 
 In that case, the operator must manually upload the certificates:
 
- 1. Store the certificates in the `credentials-high` password store, using
-    this naming convention for the key: `certs/${DEPLOY_ENV}/${CERT_NAME}.crt`
-    and `certs/${DEPLOY_ENV}/${CERT_NAME}.key`.
+ 1. Store the certificates in the `credentials-high` password store, using this
```

Comment:
It's https://github.gds/government-paas/credentials-high which is an alternative to https://github.gds/government-paas/credentials that only Civil Servants with Security Clearance have access to. It contains some more sensitive credentials, such as AWS and Pingdom root accounts.

It might be useful to make `credentials-high` a hyperlink. Even if people can't access it, they will be able to determine that it's a repo (rather than a script or service). Welcome any other suggestions too, @combor ?